pyoverkiz/overkiz-root-ca-2048.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Where did you get it?

Reply:
From https://ca.overkiz.com/overkiz-root-ca-2048.crt.